flink-runtime-web/web-dashboard/src/app/share/customize/task-manager-item/task-manager-item.component.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { TaskmanagersItemCalInterface } from 'interfaces';
+
+@Component({
+  selector: 'flink-task-manager-item',
+  templateUrl: './task-manager-item.component.html',
+  styleUrls: ['./task-manager-item.component.less'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TaskManagerItemComponent {
+  @Input() state: TaskmanagersItemCalInterface;
+  @Input() totalCount: number;
+  @Input() timeoutThresholdSeconds: number;
+  totalCountStyleThreshold: number;
```

Comment:
```suggestion
  totalCountStyleThreshold = 21;
```

Reply:
it does not work for me, when putting it here. i had to put it in the constructor to make it work. or what else am i missing?

Reply:
@drummerwolli what is the error when you putting it here? typescript error or something else?

Reply:
@vthinkxie no error message, the value is never set and is empty.

Reply:
it works for me when I test it, you can add `{{totalCountStyleThreshold}}` in the template to check if it has desired value

Reply:
yep, i did that. but no value there. and the icons are too small as well, since the evaluation does not work properly ...

Reply:
could it be something else on my side, that breaks this somehow?

Reply:
could you just try to create another test component that only including `totalCountStyleThreshold = 21;` and `{{totalCountStyleThreshold}}`, it is strange

Reply:
Hi, @drummerwolli any update on this?